examples/open_ephys_psychopy_example.ipynb

```diff
@@ -0,0 +1,233 @@
+{
```

Comment:
It is really good to have notebooks with examples, but this notebook is not well documented and has some errors at the end. If it was used for testing, please remove it, otherwise some more documentation.

Reply:
We can actually remove this notebook. The package no longer has the respective functionality. Your notebooks should suffice